packages/react-langgraph/src/useLangGraphMessages.ts

```
@@ -138,6 +138,24 @@ export const useLangGraphMessages = <TMessage extends { id?: string }>({
             break;
           case LangGraphKnownEventTypes.Error:
             onError?.(chunk.data);
+            // Update the last AI message with error status
+            // Assumes last AI message is the one the error relates to
+            const messages = accumulator.getMessages();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)